mesop/components/input/input.ts

```diff
@@ -35,6 +36,14 @@ export class InputComponent {
         debounceTime(150),
       )
       .subscribe((event) => this.onInputDebounced(event));
+    // See keyDown event for explanation for why we have a special case for Safari.
+    if (
+      navigator.userAgent.indexOf('Safari') > -1 &&
+      navigator.userAgent.indexOf('Chrome') === -1
```

Comment:
It's probably worth an inline comment to explain why you're checking for the absence of chrome (i.e. Chrome's user-agent basically spoofs other browsers and includes Safari, so we need to explicitly look for the absence of it).